applications/sasquatch/charts/app-metrics/values.yaml

```diff
@@ -0,0 +1,114 @@
+## Default values.yaml for the Metrics Events subchart.
+
+# -- app-metrics configuration in any environment in which the subchart is
+# enabled. This should stay globally specified here, and it shouldn't be
+# overridden. Looks like this:
+#
+# globalAppConfig:
+#   some-app:
+#     influxTags:
+#       - some
+#       - tags
+#   some-other-app:
+#     influxTags:
+#       - some
+#       - tags
+#       - and
+#       - others
+globalAppConfig: {}
```

Comment:
These big helm-docs comments kind of break the Markdown table, which is oriented around single paragraph descriptions. I suppose we could [look into HTML tables](https://github.com/norwoodj/helm-docs#advanced-table-rendering), but in the meantime I wonder if we could link out to documentation for this structure elsewhere? Like in the Sasquatch docs?